Gems/AWSGameLift/Code/AWSGameLiftServer/Tests/AWSGameLiftServerManagerTest.cpp

```diff
@@ -735,6 +744,68 @@ R"({
         EXPECT_TRUE(actualResult[0].m_playerAttributes.size() == 4);
     }
 
+    TEST_F(GameLiftServerManagerTest, GetExternalSessionCertificate_CallWithTLSEnabled_GetExpectedResult)
```

Comment:
can we also add tests for the new error cases (unable to locate files)?

Reply:
Have a test case when instance certificate outcome is not success, can you clarify what is unable to locate file case you want to test?

Reply:
oh, do you mean the "EmptyResult" tests? sorry, I didn't realize that's what those were covering

Reply:
yes